### ISSUE TITLE
Installation?

### ISSUE THREAD

**Issue by Hexaae** [segrax/directory_opus_amiga_plugin#3]
Please, add a few lines on the home page on how to install this with DOpus...

**Comment by rrreese** [comment]
You need to copy the file VFSPlugins\opusADF.dll

into C:\Program Files\GPSoftware\Directory Opus\VFSPlugins

Then restart Directory Opus (make sure you exit it from the system tray)

**Comment by Hexaae** [comment]
Yes. Found it by myself the same day. However Installation instructions should be added on the home of the github project IMHO
